collections/_article/maintenance-release-godot-3-6-1.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "Maintenance release: Godot 3.6.1"
+excerpt: "Godot 3.6 was released in September last year, and it has been mostly stable, although there have been a few more important bugs which warrant a maintenance \"patch\" release (3.6.x)."
+categories: ["release"]
+author: lawnjelly
+image: /storage/blog/covers/maintenance-release-godot-3-6-1.webp
+image_caption_title: CraftCraft
+image_caption_description: A game by Placeholder Gameworks
+date: 2025-06-25 12:00:00
+---
+
+[Godot 3.6](/article/godot-3-6-finally-released) was released in September last year, and it has been mostly stable, although there have been a few more important bugs which warrant a maintenance "patch" release (3.6.x).
+
+As well as bug fixes, this release also contains some cherry-picks from 4.x to update libraries. Let us know if you spot any regressions.
+
+[Jump to the **Downloads** section.](#downloads)
+
+As usual, you can try it live with the [**online version of the Godot editor**](https://editor.godotengine.org/releases/3.6.1/) updated for this release.
```

Comment:
This link is also not working currently, maybe we haven't built online version yet (and it might need to be 3.6.1-stable rather than 3.6.1).